totalContributors.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+import pathlib
+import os.path
+import sys
+import datetime
+
+def add_log(dict, input):
+    for dataset in input:
+        name = dataset.split("\t")[1]
+        value = int(dataset.split("\t")[0])
+        if name in dict:
+            dict[name] += value
+        else:
+            dict[name] = value
+
+def retrieve_log(repo, version):
+    return subprocess.run(['git', 'shortlog', '-s', version], capture_output = True, text = True, 
+        cwd = os.path.dirname(pathlib.Path().resolve()) + repo).stdout.splitlines()
+
+def find_version(time, repo):
+    versions_list = subprocess.run(['git', 'tag', '-l'], capture_output = True, text = True, 
+        cwd = os.path.dirname(pathlib.Path().resolve()) + repo).stdout.splitlines()
+    for version in versions_list:
+        version_time = subprocess.run(['git', 'log', '-1', '--format=%ai', version], capture_output = True, text = True, 
+            cwd = os.path.dirname(pathlib.Path().resolve()) + repo).stdout.split()[0]
+        if(create_Time_Object(version_time) >= create_Time_Object(time)):
+            return version
+    return version
+
+def create_Time_Object(time):
+    time_arr = time.split("-")
+    year = int(time_arr[0])
+    month = int(time_arr[1])
+    day = int(time_arr[2])
+    return datetime.datetime(year, month, day)
+    
+#extract git version and time
+if len(sys.argv) > 1:
+    zulip_version = sys.argv[1]
+else:
+    zulip_version = str(4.3)
+
+time = subprocess.run(['git', 'log', '-1', '--format=%ai', zulip_version], capture_output = True, text = True).stdout.split()[0]
+
+#retrieve versions
+zulip_mobile_version = find_version(time, "/zulip-mobile")
```

Comment:
The slash prefix makes it look like it is a filesystem path relative to the root. Maybe you should modify `find_version` to automatically add `/`, and so have the argument to be as clean as just "zulip-mobile".